deprecation-index.yml

```diff
@@ -162,18 +162,18 @@
     - EntityGetFormDisplayStatic.php
 'file_default_scheme()':
   Rector: FileDefaultSchemeRector.php
-  PHPStan: 'Call to deprecated function file_default_scheme(). Deprecated in drupal:8.8.0 and is removed from drupal:9.0.0. Use Drupal::config('system.file')->get('default_scheme') instead.'
+  PHPStan: "Call to deprecated function file_default_scheme(). Deprecated in drupal:8.8.0 and is removed from drupal:9.0.0. Use Drupal::config('system.file')->get('default_scheme') instead."
```

Comment:
There was single quote inside here so changed to using double quote so we don't have escape

Reply:
Thanks, @tedbow.

I'm going to try and write a quick Behat test for this so it doesn't happen again. This isn't the first time. :(

Can I update your PR directly to use `''` for escaping? I found that to be more affective since so many of these have `\` and other characters that makes `"` quotes kind of a mess.

Reply:
Sure yeah update it however you see fit.